src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,58 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the cluster will be encrypted while the internal components
+of Deis will still communicate over HTTP.
+
+To enable SSL, you will need to open port 443 on the load balancer and forward it to port 80 on the
+routers. For AWS, you'll also need to add port 443 in the security group settings for your load
+balancer.
+
```

Comment:
> I'd like to use the host port, however I feel like we cannot rely on this behaviour. For example, what if a user's application (or another third party app) binds to port 80 on the host? Given that there's enough nodes to schedule that pod, a load balancer pointing at the minion's host port 80 would then cycle this user's app into the routing table because it is listening on port 80, passing the health check.

That is an excellent point, but there is also a confluence of three specific things required to make that happen-- 1. the app would have to bind to host port 80, 2. it would have to bind to host port 9090 3. it would have to use `:9090/healthz` for the health check.  I figured this combination of things was extraordinarily unlikely primarily on account of the fact that user applications are almost guaranteed to listen for health checks on the same port on which they listen for everything else.  The only reason the router splits health checks out into a separate port is because I had to provide a listener on the Nginx end that was guaranteed to _never_ use PROXY protocol, since ELBs, for instance, could only do TCP-based health checks with PROXY proto in play and I wanted HTTP based health checks so that we could call out the /healthz endpoint specifically.

In the end, you are correct that this _could_ happen.  Perhaps it's fair that we not _recommend_ using the router's host ports, but we ourselves should remain tuned in to the fact that it's a viable option with slightly better performance.

Ultimately, I think if your directions are just to amend the LB's 443 listener to use HTTPS on the front and the HTTPS on the back with the same back-end port as the 80 listener (which should be a node port), that will be fine.